<ui version="4.0" >
 <class>ProjectsManager</class>
 <widget class="QDockWidget" name="ProjectsManager" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>206</width>
    <height>349</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>5</hsizetype>
    <vsizetype>7</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="allowedAreas" >
   <set>Qt::LeftDockWidgetArea|Qt::NoDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <widget class="QWidget" name="backWidget" >
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>7</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle" >
    <string>Projects Manager</string>
   </property>
   <property name="windowIcon" >
    <iconset/>
   </property>
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>3</number>
    </property>
    <property name="spacing" >
     <number>3</number>
    </property>
    <item>
     <widget class="QFrame" name="fActions" >
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>0</number>
       </property>
       <item>
        <widget class="QToolBar" name="tbActions" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>1</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="iconSize" >
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="sProjects" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>7</hsizetype>
        <vsizetype>7</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTreeWidget" name="twProjects" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>7</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="columnCount" >
        <number>1</number>
       </property>
       <column>
        <property name="text" >
         <string>Projects List</string>
        </property>
       </column>
      </widget>
      <widget class="QStackedWidget" name="swProjects" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>7</hsizetype>
         <vsizetype>7</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../resources/resources.qrc" />
 </resources>
 <connections/>
</ui>
